spec/02-integration/01-helpers/03-http_mock_spec.lua

```diff
@@ -220,3 +221,47 @@ describe("http_mock config", function()
     assert(pl_file.access_time(pid_filename) ~= nil, "mocking not in the correct place")
   end)
 end)
+
+describe("#http_mock.tapping", function()
+  local tapped, tapped_port
+  lazy_setup(function()
+    tapped, tapped_port = http_mock.new()
+    tapped:start()
+  end)
+  lazy_teardown(function()
+    tapped:stop(true)
+  end)
+
+  it("it works", function()
```

Comment:
the idea of the test framework is to make it easy to read; to the `it(` should not be repeated. The description will be added to the describe description; eg. `http_mock.tapping it works`

```suggestion
  it("works", function()
```

would create a test description `http_mock.tapping works`, which is how the framework was intended